public/app/features/trails/MetricScene.tsx

```diff
@@ -113,13 +116,39 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
     this.publishEvent(new OpenEmbeddedTrailEvent(), true);
   };
 
+  public getLinkToExplore = async () => {
+    const metricScene = sceneGraph.getAncestor(this, MetricScene);
+    const trail = getTrailFor(this);
+    const dsValue = getDataSource(trail);
+
+    const flexItem = metricScene.state.body.state.children[0] as SceneFlexItem;
+    const autoVizPanel = flexItem.state.body as AutoVizPanel;
+    const queries = autoVizPanel.state.queryDef?.queries || [];
```

Comment:
```suggestion
    const autoQuery = getAutoQueriesForMetric(metric);
    const queries = autoQuery.main.queries;
```

Reply:
ah, did not think of variants. We should probably move the variant selection out the MetricScene (at least the state), and make it part of url state